### PR TITLE
Add bc for computing resources required by qeapp image

### PR DESCRIPTION
**Pull request by unkcpz**

### CODE REVIEW

**Comment by unkcpz** [comment]
Hi @danielhollas, why the test is going to upload to registry? Should I open PR from origin repo?

File: stack/base/Dockerfile [hunk 1]
```diff
@@ -10,7 +10,8 @@ USER root
 # https://pymatgen.org/installation.html#installation-tips-for-optional-libraries
 # rsync: needed to support the new AiiDA backup command
 # povray: rendering engine used in aiidalab-widgets-base
-ENV EXTRA_APT_PACKAGES "curl povray rsync build-essential"
+# bc: needed to compute the resources for computer setup
```
Comment (danielhollas) [inline stack/base/Dockerfile:13]: Is this really needed? We could always use a Python one liner.

**Review by danielhollas — COMMENTED** [review]
I don't know why the test failed. Maybe try restarting it? 

I don't understand the question, your PR is open from origin, so it should be fine to push to ghcr.io

File: stack/base/Dockerfile (hunk 1 above)
Comment (unkcpz) [reply stack/base/Dockerfile:13]: I don't want to use python, bc is a tiny thing and all computation just happened in shell script.

**Comment by danielhollas** [comment]
Ah, sorry, I am on a phone app and it did not show me that this PR is from a fork. 

Yes, for this repo you always have to open the PRs from the origin.

**Comment by unkcpz** [comment]
Replaced by #499

**Pull request closed without merging**